tests/unit/store/entity/mutations.spec.ts

```diff
@@ -53,4 +57,14 @@ describe( 'entity/mutations', () => {
 
 	} );
 
+	it( EDITABILITY_UPDATE, () => {
+		const store = newMinimalStore( { isEditable: true } );
```

Comment:
No harm done but the default state here is the opposite of what is the case in the real store.

Reply:
changed!